src/ast/irbuilderbpf.cpp

```diff
@@ -865,6 +865,108 @@ Value *IRBuilderBPF::CreateStrncmp(Value *val1,
   return result;
 }
 
+Value *IRBuilderBPF::CreateStrstr(Value *val1,
+                                   Value *val2,
+                                   uint64_t n1,
+                                   uint64_t n2,
+                                   bool inverse)
+{
+
```

Comment:
can you add a little pseudo code description of the algorithm you're using here? It would make reviewing a bit easier :)

Reply:
Okay, I'll add it